### PR TITLE
Add Foundry and Visual Auditor

### PR DESCRIPTION
**Pull request by 0xsomnus**
Added solidity visual auditor for VSCode and Foundry, a new ETH dev framework

### CODE REVIEW

File: docs/resources.rst [hunk 1]
```diff
@@ -24,6 +24,9 @@ Integrated (Ethereum) Development Environments
 
     * `Dapp <https://dapp.tools/>`_
         Tool for building, testing and deploying smart contracts from the command line.
+    
+    * `Foundry <https://github.com/gakonst/foundry>`_
```
Comment (ekpyron) [inline docs/resources.rst:28]: The list is alphabetical, so if we want to add it, we should move it past embark.

**Review by ekpyron — COMMENTED** [review]
The tests indicate that there is some trailing whitespace in the changes that would have to be removed.

File: docs/resources.rst [hunk 2]
```diff
@@ -81,6 +84,9 @@ Editor Integrations
 
     * `Visual Studio Code extension <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
         Solidity plugin for Microsoft Visual Studio Code that includes syntax highlighting and the Solidity compiler.
+        
+    * `Solidity Visual Auditor extension <https://marketplace.visualstudio.com/items?itemName=tintinweb.solidity-visual-auditor>`_
+        Ethereum Solidity Language customized for developers and auditors using Visual Studio Code
```
Comment (cameel) [inline docs/resources.rst:89]: This sounds really confusing to me. It says it's a language. I know this line was copied from the original description but it I don't think it explains anything. Is it an extension that adds language support to VSCode? Or a language server?

**Comment by leonardoalt** [comment]
@morphean-sec it would be great if you could apply the reviewers' comments

**Comment by leonardoalt** [comment]
Not sure when that happened, but Foundry is already on the `develop` list of frameworks

**Comment by leonardoalt** [comment]
Closing in favor of https://github.com/ethereum/solidity/pull/12683 which fixes this PR keeping the author's credits, since I couldn't push directly to this branch.

**Pull request closed without merging**